### ISSUE TITLE
[BE] add documentation for adjust learning rate when going to distributed training

### ISSUE THREAD

**Issue by wanchaol** [pytorch/pytorch#75129]
### 🚀 The feature, motivation and pitch

In our doc, it's better that we could add a note in the doc for adjusting learning rate when going from one to multiple, there're some issues that user confused about adjusting it, i.e. https://discuss.pytorch.org/t/how-to-scale-warmup-the-learning-rate-for-large-batch-size/146519/3
